elasticjob-executor/elasticjob-executor-type/elasticjob-http-executor/src/main/java/org/apache/shardingsphere/elasticjob/http/executor/HttpJobExecutor.java

```diff
@@ -116,6 +118,10 @@ private boolean isWriteMethod(final String method) {
         return Arrays.asList("POST", "PUT", "DELETE").contains(method.toUpperCase());
     }
     
+    private boolean isRequestSucceed(final int httpStatusCode) {
+        return HttpURLConnection.HTTP_BAD_REQUEST > httpStatusCode;
```

Comment:
Why not use 'HttpURLConnection.HTTP_OK == httpStatusCode' to accurate the result ?

Reply:
This condition is refer to `sun.net.www.protocol.http.HttpURLConnection#getErrorStream`, which implemented `java.net.HttpURLConnection`.